changes_reviewer.go

```diff
@@ -25,6 +25,15 @@ type AddReviewerResult struct {
 	Confirm   bool           `json:"confirm,omitempty"`
 }
 
+// DeleteVoteInput  entity contains options for the deletion of a vote.
```

Comment:
Minor nit, there are 2 spaces before "entity".

Reply:
Thanks! Fixed in 95b11af